### PR TITLE
Task/gv 635 multiwallet disabled

### PR DESCRIPTION
**Pull request by Anchello**
As part of this task, I updated the data of wallets in withdraw and deposit popups is taken from store

### CODE REVIEW

File: packages/shared/components/wallet/components/wallet-balance/wallet-balance-buttons.js [hunk 1]
```diff
@@ -10,17 +10,27 @@ const WalletBalanceButtons = ({
...
+  handleTransfer,
+  isDepositEnabled,
+  isWithdrawalEnabled
 }) => {
   return (
     <div className="wallet-balance__buttons">
-      <GVButton onClick={handleAddFunds}>
+      <GVButton
+        onClick={handleAddFunds}
+        disabled={!isDepositEnabled && isDepositEnabled !== undefined}
```
Comment (pavelzubov) [inline packages/shared/components/wallet/components/wallet-balance/wallet-balance-buttons.js]: Maybe `isDepositEnabled===false`?

File: packages/shared/components/wallet/components/wallet-balance/wallet-balance-buttons.js [hunk 2]
```diff
@@ -10,17 +10,27 @@ const WalletBalanceButtons = ({
...
           +
         </span>
         {t("wallet-page.add-funds")}
       </GVButton>
-      <GVButton color="secondary" variant="outlined" onClick={handleWithdraw}>
+      <GVButton
+        color="secondary"
+        variant="outlined"
+        onClick={handleWithdraw}
+        disabled={!isWithdrawalEnabled && isWithdrawalEnabled !== undefined}
```
Comment (pavelzubov) [inline packages/shared/components/wallet/components/wallet-balance/wallet-balance-buttons.js]: Maybe `isWithdrawalEnabled ===false`?

File: packages/shared/components/wallet/components/wallet-list/wallet-list.js [hunk 3]
```diff
@@ -133,20 +133,28 @@ class WalletList extends Component {
...
                   >
                     <img src={ConvertIcon} alt="Convert Icon" />
                   </Chip>
                   <Chip
                     className="wallet-list__withdraw"
-                    onClick={this.handleOpenWithdrawPopup.bind(this, wallet)}
+                    onClick={this.handleOpenWithdrawPopup(wallet)}
+                    disabled={
+                      !wallet.isWithdrawalEnabled &&
+                      wallet.isWithdrawalEnabled !== undefined
```
Comment (pavelzubov) [inline packages/shared/components/wallet/components/wallet-list/wallet-list.js]: Maybe `isWithdrawalEnabled ===false`?

File: packages/shared/components/wallet/components/wallet-list/wallet-list.js [hunk 4]
```diff
@@ -133,20 +133,28 @@ class WalletList extends Component {
...
                     <img src={ArrowIcon} alt="Arrow Icon" />
                   </Chip>
                   <Chip
                     className="wallet-list__button-add-funds"
                     type="positive"
-                    onClick={this.handleOpenAddFundsPopup.bind(this, wallet)}
+                    onClick={this.handleOpenAddFundsPopup(wallet)}
+                    disabled={
+                      !wallet.isDepositEnabled &&
+                      wallet.isDepositEnabled !== undefined
```
Comment (pavelzubov) [inline packages/shared/components/wallet/components/wallet-list/wallet-list.js]: Maybe `isDepositEnabled ===false`?

Comment (Anchello) [reply packages/shared/components/wallet/components/wallet-list/wallet-list.js]: Done

File: packages/shared/components/wallet/components/wallet-list/wallet-list.js (hunk 3 above)
Comment (Anchello) [reply packages/shared/components/wallet/components/wallet-list/wallet-list.js]: Done

File: packages/shared/components/wallet/components/wallet-balance/wallet-balance-buttons.js (hunk 2 above)
Comment (Anchello) [reply packages/shared/components/wallet/components/wallet-balance/wallet-balance-buttons.js]: Done

File: packages/shared/components/wallet/components/wallet-balance/wallet-balance-buttons.js (hunk 1 above)
Comment (Anchello) [reply packages/shared/components/wallet/components/wallet-balance/wallet-balance-buttons.js]: Done

**Pull request merged**